frontend/src/components/Navbar/Navbar.jsx

```diff
@@ -38,7 +41,15 @@ export default function Navbar() {
             <div className="nav-profile">
                 {user ? (
                     <div className="logout">
```

Comment:
Super Nit: After this change, it seems that "logout" is not the most accurate class name anymore. 
The DIV became a mini profile data with a logout link next to it.